sources/indigo/export/materials/Specular.py

```diff
@@ -62,8 +60,7 @@ def get_format(self):
             'name': [self.material_name],
             'backface_emit': [str(self.material_group.indigo_material_emission.backface_emit).lower()],
             element_name: {
-                #'internal_medium_name': [ medium_name ],
-                'internal_medium_uid': [ +medium_index + 10000 ]
+                'internal_medium_uid': [ medium_index + 10 ] # seems indigo medium uid starts at 10...
```

Comment:
Hi,
Indigo medium UIDs don't necessarily start at 10.  In fact there are no specific numbering rules for UIDs - they just have to be unique.